uips/uip-7.md

```diff
@@ -0,0 +1,127 @@
+| uip | 7 |
+| - | - |
+| title | Stake Boost |
+| description | Improves decentralization by boosting rewards for small validators |
+| author | Henry de Valence ([@hdevalence](https://github.com/hdevalence)) |
+| discussions-to | <https://forum.penumbra.zone/t/uip-staking-boost/134> |
+| status | Draft |
+| type | Standards Track |
+| consensus | Yes |
+| created | 2024-12-08 |
+
+## Abstract
+
+This UIP introduces Stake Boost, a mechanism to improve validator
+decentralization by allocating a portion of staking rewards in each epoch to a
+single validator, selected uniformly at random from the active set. As all
+validators have an equal chance of being boosted, this mechanism boosts rewards
+for smaller validators relative to larger ones, incentivizing stake
+distribution across the network.
+
+## Motivation
+
+Currently, Penumbra's stake distribution is more centralized than desirable.
+While gradual decentralization has been occurring naturally and UI improvements
+have aimed to encourage this process, additional economic incentives could help
+accelerate stake distribution across the network. The Stake Boost mechanism
+provides direct economic incentives for delegators to stake with smaller
+validators, helping to create a more decentralized and robust network.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Parameters
+
+A new chain parameter `staking_boost_percent` is added to the staking
+parameters. This parameter define the percentage of new staking issuance
+allocated to the boost mechanism. 
+
+The default value for `staking_boost_percent` should be `20`, though for
+clarity this only affects newly generated parameters (e.g., for testnets);
+parameter selection on Penumbra mainnet is subject to community consensus and
+governance.
+
+### Validator Selection
+
+At the end of each epoch, one validator in the active set is chosen to be
+boosted.  To select the boosted validator, a `ChaCha20Rng` should be seeded
+with the previous block's app hash.
+
+As described below, this is an imperfect source of randomness compared to a
+real VRF, but probably good enough for this mechanism.
+
+### Reward Distribution
+
+Staking reward issuance is changed from the current logic to divert `staking_boost_percent` of rewards to the boost mechanism.
+
+TODO: fill in
```

Comment:
This should be filled in with reference to the source code, I believe the majority of the staking code would be unchanged (just changing the top line input to that computation), with some extra logic to adjust the boosted validator's exchange rate to account for the boost.